### PR TITLE
HELPS-452: Allow Loops to report host OS 

### PR DESCRIPTION
**Pull request by epiccoleman**
Adds a new System Aptitude to the LDK with an `operatingSystem` function.

### CODE REVIEW

File: docs/src/markdown-pages/aptitudes/system.operating-system.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: "Operating System"
+links_js: "operatingSystem"
+---
+Returns a string representation of the host Operating System.
+
+Possible values:
```
Comment (cshields-oliveai) [inline docs/src/markdown-pages/aptitudes/system.operating-system.md:7]: Doesn't OH support Linux still?

File: ldk/javascript/src/system/system.test.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+describe('System', () => {
+  beforeEach(() => {
+    oliveHelps.system = {
+      operatingSystem: jest.fn(),
+    };
+  });
+
+  describe('operatingSystem', () => {
+    it('returns a promise with the token', () => {
```
Comment (cshields-oliveai) [inline ldk/javascript/src/system/system.test.ts:12]: Not sure you meant "with the token"

File: ldk/javascript/src/webpack/generate-banner.ts [hunk 3]
```diff
@@ -19,6 +19,7 @@ export function generateMetadata(ldkSettings: LdkSettings): string {
       keyboard: ldkSettings.ldk.permissions.keyboard || undefined,
       network: ldkSettings.ldk.permissions.network || undefined,
       process: ldkSettings.ldk.permissions.process || undefined,
+      system: ldkSettings.ldk.permissions.system || undefined,
```
Comment (cshields-oliveai) [inline ldk/javascript/src/webpack/generate-banner.ts:22]: We will most likely have to coordinate with Loop Lib to ensure their permissions parsing can handle this new aptitude.

File: ldk/javascript/src/system/index.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { promisify } from "../promisify"
+
+
+/**
+ * The SystemService provides access to various operating system details
```
Comment (cshields-oliveai) [inline ldk/javascript/src/system/index.ts:5]: "System Aptitude"

File: ldk/javascript/src/system/system.test.ts (hunk 2 above)
Comment (epiccoleman) [reply ldk/javascript/src/system/system.test.ts:12]: Copy / paste leftovers - will fix.

File: docs/src/markdown-pages/aptitudes/system.operating-system.md (hunk 1 above)
Comment (epiccoleman) [reply docs/src/markdown-pages/aptitudes/system.operating-system.md:7]: Good question, I wasn't sure. But based on the [Installation Guide](https://oliveai.dev/guides/olive-helps-install-guide) on the Dev Hub, we do not support Linux, only macOS and Windows.

**Review by MaggieMaSS — APPROVED** [review]

**Comment by MaggieMaSS** [comment]
I think you need to run prettier :)

File: ldk/javascript/src/webpack/generate-banner.ts (hunk 3 above)
Comment (epiccoleman) [reply ldk/javascript/src/webpack/generate-banner.ts:22]: I now have a PR in progress to update the Loop Lib schemas, so I'm going to mark this resolved.

**Review by cshields-oliveai — APPROVED** [review]

**Pull request merged**